CleverPush/Source/CPStoriesController.h

```diff
@@ -10,13 +10,18 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)reloadReadStories:(NSArray *)array;
 @end
 
+@protocol storyViewOpenedListener <NSObject>
+- (void)openUrl:(BOOL)isOpened;
+@end
+
 @interface CPStoriesController : UIViewController<UIGestureRecognizerDelegate, iCarouselDataSource, iCarouselDelegate, WKScriptMessageHandler>
 
 @property (nonatomic, strong) NSMutableArray<CPStory*> *stories;
 @property (nonatomic, assign) NSInteger storyIndex;
 @property (nonatomic, strong) IBOutlet CleverPushiCarousel *carousel;
 @property (nonatomic, strong) NSMutableArray *readStories;
 @property (nonatomic, assign) id delegate;
+@property (nonatomic, assign) id<storyViewOpenedListener> openUrlDelegate;
```

Comment:
not id<storyViewOpenedListener>, just do `CPStoryViewOpenedBlock`.

It will be defined like this:

`typedef void (^CPStoryViewOpenedBlock)(URL* url);`

Reply:
see https://github.com/cleverpush/cleverpush-ios-sdk/blob/master/CleverPush/Source/CleverPushInstance.h#LL48C1-L49C1